src/Microsoft.SqlTools.ServiceLayer/Metadata/Contracts/GenerateServerContextualizationCompleteNotification.cs

```diff
@@ -0,0 +1,36 @@
+﻿//
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+//
+
+using Microsoft.SqlTools.Hosting.Protocol.Contracts;
+
+namespace Microsoft.SqlTools.ServiceLayer.Metadata.Contracts
+{
+    public class GenerateServerContextualizationCompleteParams
+    {
+        /// <summary>
+        /// The URI identifying the owner of the connection
+        /// </summary>
+        public string OwnerUri { get; set; }
+        /// <summary>
+        /// Indicates if generating context completed successfully
+        /// </summary>
+        public bool CompletedGeneratingContext { get; set; }
+        /// <summary>
+        /// Holds any error messages, if errors were encountered while
+        /// generating context
+        /// </summary>
+        public string Messages { get; set; }
```

Comment:
have Error in the name so it's clear this has error messages without needing to look at the comments

Reply:
I was thinking that it would default to an empty string, but I'm thinking that setting the owner URI is a better way to go.